.github/workflows/autoupdate-preview.yaml

```diff
@@ -56,7 +60,20 @@ jobs:
             ## Using early preview changes
 
             ```sh
-            go get go.mongodb.org/atlas-sdk@dev-latest   
+            go get github.com/mongodb/atlas-sdk-go@dev-latest   
+            ```
+
+            > NOTE: You have to run the `go get` command every time you want to update the Preview version to the latest version.
+            
+            After that you can use it as usual, e.g.:
+            ```go
+            ...
+            import(
+              adminPreview "github.com/mongodb/atlas-sdk-go/admin"
+            ...
+              		
+            params := &adminPreview.ListProjectsApiParams{ ...
```

Comment:
[Separately to PR] Please note that this will require separate SDK client initialization.

Reply:
yes, this was just an example of how to refer to the new Preview module while being able to use the regular one

Reply:
I think you could be better with example/docs as reference rather than code being present in PR description. 
It is size restricted and can lead to confusion

Reply:
that's a good point, will removed from PR description